files/fr/web/media/dash_adaptive_streaming_for_html_5_video/index.md

```diff
@@ -31,48 +31,64 @@ Dans les lignes d'exemple qui suivent, on utilise le fichier de départ **_in.vi
 
 On créera la piste audio avec :
 
-    ffmpeg -i in.video -vn -acodec libvorbis -ab 128k -dash 1 my_audio.webm
+```bash
+ffmpeg -i in.video -vn -acodec libvorbis -ab 128k -dash 1 my_audio.webm
+```
 
 On créera les pistes vidéos avec :
 
-    ffmpeg -i in.video -c:v libvpx-vp9 -keyint_min 150 -g 150 -tile-columns 4 -frame-parallel 1  -f webm -dash 1 \
-    -an -vf scale=160:90 -b:v 250k -dash 1 video_160x90_250k.webm
+```
+ffmpeg -i in.video -c:v libvpx-vp9 -keyint_min 150 -g 150 -tile-columns 4 -frame-parallel 1  -f webm -dash 1 \
+-an -vf scale=160:90 -b:v 250k -dash 1 video_160x90_250k.webm
+```
 
-    ffmpeg -i in.video -c:v libvpx-vp9 -keyint_min 150 -g 150 -tile-columns 4 -frame-parallel 1  -f webm -dash 1 \
-    -an -vf scale=320:180 -b:v 500k -dash 1 video_320x180_500k.webm
+```
+ffmpeg -i in.video -c:v libvpx-vp9 -keyint_min 150 -g 150 -tile-columns 4 -frame-parallel 1  -f webm -dash 1 \
+-an -vf scale=320:180 -b:v 500k -dash 1 video_320x180_500k.webm
+```
 
-    ffmpeg -i in.video -c:v libvpx-vp9 -keyint_min 150 -g 150 -tile-columns 4 -frame-parallel 1  -f webm -dash 1 \
-    -an -vf scale=640:360 -b:v 750k -dash 1 video_640x360_750k.webm
+```
+ffmpeg -i in.video -c:v libvpx-vp9 -keyint_min 150 -g 150 -tile-columns 4 -frame-parallel 1  -f webm -dash 1 \
```

Comment:
```suggestion
```bash
ffmpeg -i in.video -c:v libvpx-vp9 -keyint_min 150 -g 150 -tile-columns 4 -frame-parallel 1  -f webm -dash 1 \
```